Configurations/15-ios.conf

```diff
@@ -53,12 +70,10 @@ my %targets = (
     },
     "ios-cross" => {
         inherit_from     => [ "ios-xcrun" ],
-        CC               => "cc",
         cflags           => add("-isysroot \$(CROSS_TOP)/SDKs/\$(CROSS_SDK)"),
     },
     "ios64-cross" => {
         inherit_from     => [ "ios64-xcrun" ],
-        CC               => "cc",
         cflags           => add("-isysroot \$(CROSS_TOP)/SDKs/\$(CROSS_SDK)"),
     },
-);
+);
```

Comment:
Please include a new line at the end of the file.

Reply:
done.